### PR TITLE
DT Refactor

### PR DESCRIPTION
**Pull request by prykon**
Starting a theme refactor...
The goal is to tidy up code written a while back and make it easier for maintaining and new collaborators to engage with it.

### CODE REVIEW

**Comment by prykon** [comment]
@corsacca If you want, we can accumulate a few commits (each commit, a refactored file) and review them one by one.

File: dt-assets/functions/cleanup.php [hunk 1]
```diff
@@ -229,19 +224,16 @@ function dt_disable_wp_emoji() {
...
 
 function dt_disable_emojis_tinymce( $plugins ) {
+    $plugins_array = [];
     if ( is_array( $plugins ) ) {
-        return array_diff( $plugins, array( 'wpemoji' ) );
-    } else {
-        return array();
+        $plugins_array = array_diff( $plugins, array( 'wpemoji' ) );
     }
+    return $plugins_array;
```
Comment (corsacca) [inline dt-assets/functions/cleanup.php:231]: @prykon this is simpler
```
    if ( is_array( $plugins ) ) {
        return array_diff( $plugins, array( 'wpemoji' ) );
    }
    return [];
```

**Comment by prykon** [comment]
@corsacca dt-assets/functions/cleanup.php has comments on the filters inside the `dt_start()` function but then redundant comments on each of the callback functions.

**Example:**
Screen Shot 2022-08-05 at 12 10 01

Screen Shot 2022-08-05 at 12 10 33

**Should we remove...?**
- The comment on the actions/filters
- The comment on the callback function
- Both
- Neither

Or should we change the callback function names for descriptive ones that render the comments irrelevant?

File: dt-assets/functions/cleanup.php (hunk 1 above)
Comment (corsacca) [reply dt-assets/functions/cleanup.php]: @prykon why keep the redundant variable?

**Comment by corsacca** [comment]
@prykon i'd leave both.

File: dt-assets/functions/cleanup.php (hunk 1 above)
Comment (prykon) [reply dt-assets/functions/cleanup.php]: @corsacca Updated.

**Comment by corsacca** [comment]
thank @prykon

**Pull request merged**